recipes/importmagic

```diff
@@ -0,0 +1,3 @@
+(importmagic :fetcher github
+             :repo "anachronic/importmagic.el"
+             :files ("importmagic.el" "importmagicserver.py"))
```

Comment:
Use `:defaults` value instead of specifying each `.el` files.

Please read https://github.com/melpa/melpa#recipe-format